manualTests/DevicePropertiesGetAllAsync/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿// See https://github.com/vestervang/DotNet-BlueZ/issues/8
+
+using vestervang.DotNetBlueZ;
+// using HashtagChris.DotNetBlueZ;
```

Comment:
This comment should be removed

Reply:
Resolved by [6d46fd5](https://github.com/vestervang/DotNet-BlueZ/pull/10/commits/6d46fd5b884e42a559a9063aa9c3e814da0d1f9e)